modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -42,6 +49,59 @@ public class ObjectDefinitionExportImportTest extends BaseExportImportTestCase {
 	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testExportImportLocalizedObjectDefinition() throws Exception {
+
+		// Localized modifiable system object definition
+
+		testExportImport(
+			"test-modifiable-system-object-definition.portuguese-default-" +
+				"locale.json",
+			"test-modifiable-system-object-definition.site-default-locale.json",
+			"TESTMODIFIABLESYSTEMOBJECTDEFINITIONPORTUGUESE",
+			"TestModifiableSystemObjectDefinitionptBR");
+
+		objectDefinitionResource.deleteObjectDefinition(
+			getId("TestModifiableSystemObjectDefinitionptBR"));
+
+		// Localized object definition
+
+		testExportImport(
+			"test-object-definition.portuguese-default-locale.json",
+			"test-object-definition.site-default-locale.json",
+			"TESTOBJECTDEFINITIONPORTUGUESE", "TestObjectDefinitionptBR");
+
+		objectDefinitionResource.deleteObjectDefinition(
+			getId("TestObjectDefinitionptBR"));
+
+		// Localized object definition with portuguese removed from the company
+		// available locales
```

Comment:
// Localized object definition with **the** portuguese **locale** removed from the company
// available locales

Reply:
Done!